sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v1/DisplayWorkV1.scala

```diff
@@ -0,0 +1,145 @@
+package uk.ac.wellcome.display.models.v1
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+import uk.ac.wellcome.display.models._
+import uk.ac.wellcome.models.IdentifiedWork
+
+@ApiModel(
+  value = "Work",
+  description =
+    "An individual work such as a text, archive item or picture; or a grouping of individual works (so, for instance, an archive collection counts as a work, as do all the series and individual files within it).  Each work may exist in multiple instances (e.g. copies of the same book).  N.B. this is not synonymous with \\\"work\\\" as that is understood in the International Federation of Library Associations and Institutions' Functional Requirements for Bibliographic Records model (FRBR) but represents something lower down the FRBR hierarchy, namely manifestation. Groups of related items are also included as works because they have similar properties to the individual ones."
+)
+case class DisplayWorkV1(
```

Comment:
No need to dupe the version identifier in the name when namespaced?

Reply:
True, but it might be helpful with IDE autocompletion and in general to avoid errors. It's more visible in the class name

Reply:
I must admit, I'd prefer not to dupe the version in the class name as well. You can always alias the type name locally (`import *.v1.DisplayWork => V1DisplayWork`).